src/mongo/db/repl/member_config.h

```diff
@@ -100,6 +101,14 @@ class MemberConfig {
     }
 
     /**
+     * Gets the canonical internal replication name for this member.
+     This will be used for all internal replication traffic.
+     */
+    const HostAndPort& getHostInternalAndPort() const {
```

Comment:
s/getHostInternalAndPort/getInternalHostAndPort

(since the HostAndPort is the type)